src/domain/child/modal/ChildFormModal.tsx

```diff
@@ -0,0 +1,119 @@
+import Button from '../../../common/components/button/Button';
+import SelectField from '../../../common/components/form/fields/select/SelectField';
+import { CHILD_RELATIONSHIP_OPTIONS } from '../constants/ChildRelationshipConstants';
+import { Child } from '../types/ChildTypes';
+
+export interface ChildFormModalValues extends Omit<Child, 'birthdate'> {
+  birthdate: {
+    day: number | string;
+    month: number | string;
+    year: number | string;
+  };
+}
+interface ChildFormModalProps {
+  initialValues: ChildFormModalValues;
+  label: string;
+  onSubmit: (values: ChildFormModalValues) => void;
+  isOpen: boolean;
+  setIsOpen: (value: boolean) => void;
+}
+
+const ChildFormModal: React.FunctionComponent<ChildFormModalProps> = ({
+  initialValues,
+  label,
+  onSubmit,
+  isOpen,
+  setIsOpen,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <div className={styles.childFormModalWrapper}>
+      <Modal isOpen={isOpen} label={label} toggleModal={setIsOpen}>
+        <Formik initialValues={initialValues} onSubmit={onSubmit}>
+          {({ isSubmitting, isValid, handleSubmit }) => (
+            <form onSubmit={handleSubmit}>
+              <FieldArray
+                name="birthdate"
+                render={props => <BirthdateFormField {...props} />}
+              />
+
+              <div className={styles.childInfo}>
+                <EnhancedInputField
+                  className={styles.childHomeCity}
+                  name="homeCity"
+                  label={t(
+                    'homePage.preliminaryForm.childHomeCity.input.label'
+                  )}
+                  required={true}
+                  component={InputField}
+                  placeholder={t(
+                    'homePage.preliminaryForm.childHomeCity.input.placeholder'
+                  )}
+                />
+
+                <EnhancedInputField
+                  className={styles.childPostalCode}
+                  name="postalCode"
+                  label={t('registration.form.child.postalCode.input.label')}
+                  component={InputField}
+                  placeholder={t(
+                    'registration.form.child.postalCode.input.placeholder'
+                  )}
+                />
+              </div>
+
+              <div className={styles.childName}>
+                <EnhancedInputField
+                  name="firstName"
+                  label={t('registration.form.child.firstName.input.label')}
+                  component={InputField}
+                  placeholder={t(
+                    'registration.form.child.firstName.input.placeholder'
+                  )}
+                />
+                <EnhancedInputField
+                  name="lastName"
+                  label={t('registration.form.child.lastName.input.label')}
+                  component={InputField}
+                  placeholder={t(
+                    'registration.form.child.lastName.input.placeholder'
```

Comment:
As we discussed: It is probably best to not prefill this field as it might be quite confusing for the parent

Reply:
Done